### PR TITLE
FAQ - Non-CWA QR Codes

### PR DESCRIPTION
**Pull request by heinezen**
Fixes https://github.com/corona-warn-app/cwa-website/issues/797

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -133,13 +133,15 @@
...
                             "<b>The code was already used once for registration</b>",
                             "Every QR code you receive can only be entered into the app once. If you use the Corona Warn App on multiple devices, the code can only be registered in one device.",
                             "If you have already registered the QR code, but then deleted it from the app, the code can also not be entered a second time.",
+                            "<b>The code you received is designed for another app or service</b>",
+                            "QR codes for the Corona Warn App are distributed on a standardized information sheet as part of form 10C. You can compare the QR code information you received with the <a href='https://www.kbv.de/media/sp/Ansichtsexemplar_Muster10C.pdf' target='_blank' rel='noopener noreferrer'>bottom half of this example form</a>. If you have not received the depicted information sheet, your QR code is likely incomptable to the app. In this case, please contact the doctor's office or testing center where you made the test and ask them where your QR code can be registered.",
```
Comment (MikeMcC399) [inline src/data/faq.json:141]: incomptable => incompatible

File: src/data/faq_de.json [hunk 2]
```diff
@@ -44,1240 +44,1242 @@
...
+                            "Hinweis: Wie oben beschrieben: Aktivieren Sie die allgemeine Systemeinstellung 'Standort verwenden' unter 'Einstellungen' > 'Sicherheit & Standort' > 'Standort', da sonst der Austausch der verschlüsselten Zufalls-IDs zur Risiko-Ermittlung nicht funktioniert."
+                        ]
+                    },
+                    {
+                        "title": "Beim Einscannen des QR Codes erhalte ich einen Fehler. Was kann ich tun?",
+                        "anchor": "qr_scan",
+                        "active": true,
+                        "textblock": [
+                            "Dieses Problem kann zwei verschiedene Ursachen haben:",
+                            "<b>Der QR Code wurde bereits einmal zur Registrierung benutzt</b>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:137]: In the line above this one
  "Dieses Problem kann zwei verschiedene Ursachen haben:",

"zwei" needs to be changed to "drei"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Just two minor corrections for you.

File: src/data/faq_de.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq_de.json:137]: ok

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json]: ok

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks.

**Comment by dsarkar** [comment]
@heinezen,  @MikeMcC399 

We should add these sentences:
 
- Alle nicht-notwendigen persönlichen Daten sollten geschwärzt werden. 
 
- All non-essential personal data should be blackened. 
 
 
 I suggest as the second last sentence.

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
EN
image

**Comment by dsarkar** [comment]
DE
image

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@heinezen Review, and merge, unless any further suggestions/objections from the community. Thanks!
CC @MikeMcC399

**Comment by MikeMcC399** [comment]
@dsarkar 
The patient portion of the form https://www.kbv.de/media/sp/Ansichtsexemplar_Muster10C.pdf only contains the name of the person being tested and the QR code. If somebody sends e-mail to corona-warn-app.opensource@sap.com they are going to be personally identified by their sender address anyway. So there is probably little personal information, if any, that might need to be removed.

The statement requesting removal of unnecessary information does however put you on the safe side.

**Comment by dsarkar** [comment]
@MikeMcC399 Agreed!

File: src/data/faq.json [hunk 3]
```diff
@@ -133,13 +133,15 @@
...
+                            "In some cases, the error can occur during the first registration of the QR Code. This can happen if a test center generated the QR code in a wrong format. We are currently looking into how to improve the app to detect wrongly generated QR Codes and still accept them. You can find more details about the errors <a href='https://github.com/corona-warn-app/cwa-documentation/issues/400' target='_blank' rel='noopener noreferrer'>on GitHub</a>.",
+                            "If you are sure that the QR code was not scanned before and you received the correct information sheet, but are still receiving errors, you can also report this case by mail to <a href='mailto:corona-warn-app.opensource@sap.com'>corona-warn-app.opensource@sap.com</a>. Please provide a scan of the information sheet with the QR code as well as contact information of the doctor or test center where you received it. All non-essential personal data should be blackened. We will then try to resolve QR code generation issues together with the involved parties."
```
Comment (MikeMcC399) [inline src/data/faq.json:144]: blackened => blackened out

Reference: https://www.collinsdictionary.com/dictionary/english/black-out

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One suggestion for change.

File: src/data/faq.json (hunk 3 above)
Comment (heinezen) [reply src/data/faq.json]: fixed

**Pull request merged**